Library/Homebrew/dev-cmd/update-maintainers.rb

```diff
@@ -0,0 +1,64 @@
+# typed: false
+# frozen_string_literal: true
+
+require "cli/parser"
+require "utils/github"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  sig { returns(CLI::Parser) }
+  def update_maintainers_args
+    Homebrew::CLI::Parser.new do
+      description <<~EOS
+        Update the list of maintainers in the `Homebrew/brew` README.
+      EOS
+
+      named_args :none
+    end
+  end
+
+  def update_maintainers
+    update_maintainers_args.parse
+
+    # We assume that only public members wish to be included in the README
+    public_members = GitHub.public_member_usernames("Homebrew")
+
+    plc = GitHub.members_by_team("Homebrew", "plc")
+    tsc = GitHub.members_by_team("Homebrew", "tsc")
+    linux = GitHub.members_by_team("Homebrew", "linux")
+    other = GitHub.members_by_team("Homebrew", "maintainers")
+    other.except!(*[plc, tsc, linux].map(&:keys).flatten.uniq)
+
+    sentences = [plc, tsc, linux, other].map do |h|
+      h.slice!(*public_members)
+      h.each { |k, v| h[k] = "[#{v}](https://github.com/#{k})" }
```

Comment:
Can you use something other than `k, v` to clarify what the keys and values are?